Packs/ShiftLeft/Integrations/shiftleft/shiftleft.py

```diff
@@ -0,0 +1,338 @@
+                "id",
+                "app",
+                "language",
+                "version",
+                "started_at",
+                "completed_at",
+                "number_of_expressions",
+            ],
+        )
+        markdown += "\n### Recommendation\n"
+        if reachable_oss_found:
+            markdown += "\nFindings with attacker-reachable opensource vulnerabilities requires your attention!"
+        else:
+            markdown += "\nNo findings with attacker-reachable opensource vulnerabilities was found."
+        markdown += f"\n### ShiftLeft Findings ({len(filtered_findings)})\n"
+        markdown += tableToMarkdown(
+            "",
+            filtered_findings,
+            headers=[
+                "id",
+                "severity",
+                "title",
+                "file_locations",
+                "cves",
+            ],
+        )
+        return CommandResults(
+            readable_output=markdown,
+            outputs_prefix="ShiftLeft",
+            outputs_key_field="scan.id",
+            outputs=response,
+        )
+    else:
+        return CommandResults(
+            outputs_prefix="ShiftLeft",
+            outputs_key_field="",
+            outputs={},
+        )
+
+
+""" MAIN FUNCTION """
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    org_id = demisto.params().get("org_id")
+    access_token = demisto.params().get("access_token")
+    if not org_id or not access_token:
+        return_error("Both organization id and access token must be set")
+    # get the service API url
+    base_url = "https://www.shiftleft.io/api/v4"  # disable-secrets-detection
+
+    verify_certificate = True
+
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
+    proxy = demisto.params().get("proxy", False)
+
+    demisto.debug(f"Command being called is {demisto.command()}")
+    try:
+        headers: Dict = {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {access_token}",
+        }
+
+        client = ShiftLeftClient(
+            base_url=base_url, verify=verify_certificate, headers=headers, proxy=proxy
+        )
+
+        if demisto.command() == "test-module":
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client, org_id)
+            return_results(result)
+
+        elif demisto.command() == "shiftleft-list-app-findings":
+            return_results(list_app_findings_command(client, org_id, demisto.args()))
+
+        elif demisto.command() == "shiftleft-list-app-secrets":
+            return_results(list_app_secrets_command(client, org_id, demisto.args()))
+
+        elif demisto.command() == "shiftleft-list-apps":
+            return_results(list_apps_command(client, org_id, demisto.args()))
```

Comment:
Please save the `command` in a variable once to prevent loading the server configuration each time, it is more efficient. For example:
```suggestion
        command = demisto.command()
        if command == "test-module":
            # This is the call made when pressing the integration Test button.
            result = test_module(client, org_id)
            return_results(result)

        elif command == "shiftleft-list-app-findings":
            return_results(list_app_findings_command(client, org_id, demisto.args()))

        elif command == "shiftleft-list-app-secrets":
            return_results(list_app_secrets_command(client, org_id, demisto.args()))

        elif command == "shiftleft-list-apps":
            return_results(list_apps_command(client, org_id))
```